VideoApp/VideoApp/Video/LocalMediaController.m

```diff
@@ -119,13 +121,22 @@ - (void)destroyLocalVideoTrack {
 }
 
 - (void)flipCamera {
-    [self.camera flipCamera];
+    [self.camera flipCamera:self.useMultipartyMedia];
 }
 
 - (BOOL)shouldMirrorLocalVideoView {
     return self.camera.shouldMirrorLocalVideoView;
 }
 
+- (void)checkVideoSenderSettings:(BOOL)isMultiparty {
+    if (SwiftToObjc.isVideoCodecVP8Simulcast && self.localParticipant != nil && isMultiparty != self.useMultipartyMedia) {
+        self.useMultipartyMedia = isMultiparty;
+        TVIEncodingParameters *encodingParameters = [[TVIEncodingParameters alloc] initWithAudioBitrate:0 videoBitrate:1800];
+        [self.localParticipant setEncodingParameters:encodingParameters];
```

Comment:
This came over from MultiPartyExample but I'm thinking it isn't needed. Chris suggested only changing the frame rate dynamically. How is setting encoding parameters on local participant different from setting encoding parameters on connect options?